### ISSUE TITLE
NPE for io.getstream.chat.android.client.models.User.getId()

### ISSUE THREAD

**Issue by bibibilu** [GetStream/stream-chat-android#1313]
I am using the latest 4.5.2.
I am using a custom ChannelViewHolderFactory, and my ChannelsView is bound to ChannelsViewModel: 
```
private val filter = and(
          eq("type", "messaging"),
          `in`("members", listOf(ChatDomain.instance().currentUser.id))
  )
private val sort = ChannelsViewModel.DEFAULT_SORT
private val limit = 100
private val viewModel: ChannelsViewModel by viewModels { ChannelsViewModelFactory(filter, sort, limit) }

viewModel.bindView(binding.chatChannelsView, viewLifecycleOwner)
binding.chatChannelsView.setViewHolderFactory(myChannelsViewHolderFactory)
```
Every time I receive new message in the foreground, it crashes for this exception:
```
E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-1
    Process: ai.zira.employee, PID: 29649
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String io.getstream.chat.android.client.models.User.getId()' on a null object reference
        at io.getstream.chat.android.livedata.EventHandlerImpl.enrichWithOwnReactions(EventHandlerImpl.kt:463)
        at io.getstream.chat.android.livedata.EventHandlerImpl.updateOfflineStorageFromEvents$stream_chat_android_offline_release(EventHandlerImpl.kt:228)
        at io.getstream.chat.android.livedata.EventHandlerImpl$updateOfflineStorageFromEvents$1.invokeSuspend(Unknown Source:12)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

```
I did not encounter this issue in the previous version before 4.5.2. Do you have any ideas? Thanks!

**Comment by bychkovdmitry** [comment]
@bibibilu 
Thanks for reporting this. Hopefully the crash will be fixed after this PR is merged: https://github.com/GetStream/stream-chat-android/pull/1335 We will try to release the new version with the fix ASAP (4.5.3).

**Comment by bibibilu** [comment]
It works! Thank you!
